driver-reactive-streams/build.gradle

```diff
@@ -61,9 +61,11 @@ ext {
 
 jar {
     bnd (
-        '-exportcontents': "*;-noimport:=true",
+        '-exportcontents': "*",
```

Comment:
I have reverted this so that I set `-noimport:=true` and explicitly set the `Import-Package` list.